cdap-ui/app/cdap/components/FastAction/SendEventAction/SendEventAction.less

```diff
@@ -36,50 +36,45 @@
         height: 300px;
         padding: 10px;
 
-        textarea.form-control {
-          border: 1px solid #cccccc;
-        }
+        .wrangler-plus-button {
+          width: 98%;
+          height: 80%;
```

Comment:
I don't believe we need to position the elements. We just position the existing `wrangler-plus-button` relative and remove all `top, left, bottom` positions and just have height set for the dropdown container and the send button container.